qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1229,8 +1254,15 @@ def add_nodes_from(layer, nodes):
                 return
 
             # Construct a shallow copy of self
-            new_layer = copy.copy(self)
-            new_layer.multi_graph = nx.MultiDiGraph()
+            # new_layer = copy.copy(self)
```

Comment:
Can these comments be removed?

Reply:
Removed in 1345988b35c8274f2ad42a266e77e8e061ac11f7 .